src/CMakeLists.txt

```diff
@@ -178,12 +181,9 @@ if (OpenMVG_USE_OPENMP)
       endif()
     endif()
     list(APPEND OPENMVG_LIBRARY_DEPENDENCIES OpenMP::OpenMP_CXX)
+  else (OPENMP_FOUND)
+      UPDATE_CACHE_VARIABLE(OpenMVG_USE_OPENMP OFF)
   endif (OPENMP_FOUND)
-else (OpenMVG_USE_OPENMP)
-    option(OpenMVG_USE_OPENMP "Use OpenMP for parallelization" OFF)
-    include(UpdateCacheVariable)
-    UPDATE_CACHE_VARIABLE(OpenMVG_USE_OPENMP OFF)
-    remove_definitions(-DOPENMVG_USE_OPENMP)
```

Comment:
This definition never gets added if OpenMP isn't found, so no need to remove it.